handlers/transcode.go

```diff
@@ -74,30 +73,6 @@ func (d *CatalystAPIHandlersCollection) TranscodeSegment() httprouter.Handle {
 	}
 }
 
-// stream from a source to a destination buffer while also printing
-func streamOutput(src io.Reader, dst *bytes.Buffer, out io.Writer) error {
-	mw := io.MultiWriter(dst, out)
-	s := bufio.NewReader(src)
-	for {
-		var line []byte
-		line, err := s.ReadSlice('\n')
-		if err == io.EOF && len(line) == 0 {
-			break
-		}
-		if err == io.EOF {
-			return fmt.Errorf("Improper termination: %v", line)
-		}
-		if err != nil {
-			return err
-		}
-		_, err = mw.Write(line)
-		if err != nil {
-			return err
-		}
-	}
-	return nil
-}
```

Comment:
@emranemran moved this to `logging.go`. Please check for sanity. Removed multiwriter - deemed unnecessary